openapi/docs.go

```diff
@@ -0,0 +1,33 @@
+// Package classification Spinup API.
+//
+// API definitions for Spinup
+//
+//     Schemes: http, https
+//     BasePath: /
+//     Version: 0.9.1
+//     Host: localhost:4434
+//
+//     Consumes:
+//     - application/json
+//
+//     Produces:
+//     - application/json
```

Comment:
I remember you mentioned during the last standup that there are a few endpoints that aren't returning `application/json`. Do we need to get that fixed before @idoqo?

Reply:
yeah right, creating an issue for that now.